projects/1.20/assets/sculky-bits/sculkybits/lang/zh_cn.json

```
@@ -0,0 +1,49 @@
+{
+    "entity.sculkybits.sculk_saprophyte": "幽匿腐生者",
+    "item.sculkybits.sculk_saprophyte_spawn_egg": "幽匿腐生者生成蛋",
+    "entity.sculkybits.sculk_crawler": "幽匿爬行者",
+    "item.sculkybits.sculk_crawler_spawn_egg": "幽匿爬行者生成蛋",
+    "block.sculkybits.sculk_jaw": "幽匿颚",
+    "block.sculkybits.sculk_flesh": "幽匿肉",
+    "block.sculkybits.sculk_growth": "幽匿生长",
+    "block.sculkybits.sculk_cage": "幽匿牢笼",
+    "block.sculkybits.sculk_trap": "幽匿陷阱",
+    "block.sculkybits.sculk_lurker": "幽匿潜伏者",
+    "block.sculkybits.sculk_ribs": "幽匿肋骨",
+    "block.sculkybits.sculk_bones": "幽匿骨骼",
+    "block.sculkybits.sculk_stabber": "幽匿刺刀",
+    "block.sculkybits.sculk_absorber": "幽匿吸收者",
+    "block.sculkybits.sculk_lurer": "幽匿诱饵",
+    "block.sculkybits.sculk_feeler": "幽匿触感",
+    "block.sculkybits.sculk_nest": "幽匿巢穴",
+    "block.sculkybits.sculk_bone_block": "幽匿骨块",
+    "effect.sculkybits.sculk": "幽匿",
+    "item.minecraft.potion.effect.sculk_potion": "幽匿药水",
+    "item.minecraft.splash_potion.effect.sculk_potion": "幽匿溅射药水",
+    "item.minecraft.lingering_potion.effect.sculk_potion": "幽匿持续药水",
+    "item.minecraft.tipped_arrow.effect.sculk_potion": "幽匿箭矢",
+    "item.minecraft.potion.effect.short_sculk_potion": "幽匿药水",
+    "item.minecraft.splash_potion.effect.short_sculk_potion": "幽匿溅射药水",
+    "item.minecraft.lingering_potion.effect.short_sculk_potion": "幽匿持续药水",
+    "item.minecraft.tipped_arrow.effect.short_sculk_potion": "幽匿箭矢",
+    "item.minecraft.potion.effect.strong_sculk_potion": "强效幽匿药水",
+    "item.minecraft.splash_potion.effect.strong_sculk_potion": "强效幽匿溅射药水",
+    "item.minecraft.lingering_potion.effect.strong_sculk_potion": "强效幽匿持续药水",
+    "item.minecraft.tipped_arrow.effect.strong_sculk_potion": "强效幽匿箭矢",
+    "item.sculkybits.sculk_bucket": "幽匿桶",
+    "death.attack.sculk": "%1$s 被幽匿夺走了",
+    "death.attack.sculk.player": "%1$s 试图逃离 %2$s 时被幽匿夺走了",
+    "death.attack.sculkAttack": "%1$s 被幽匿吸收了",
+    "death.attack.sculkAttack.player": "%1$s 试图逃离 %2$s 时被幽匿吸收了",
+    "death.attack.sculkEntityAttack": "%1$s 被 %2$s 击杀了",
+    "death.attack.sculkEntityAttack.player": "%1$s 试图逃离 %3$s 时被 %2$s 击杀了",

```

```suggestion
    "death.attack.sculk": "%1$s被幽匿夺走了",
    "death.attack.sculk.player": "%1$s在试图逃离%2$s时被幽匿夺走了",
    "death.attack.sculkAttack": "%1$s被幽匿吸收了",
    "death.attack.sculkAttack.player": "%1$s在试图逃离%2$s时被幽匿吸收了",
    "death.attack.sculkEntityAttack": "%1$s被%2$s击杀了",
    "death.attack.sculkEntityAttack.player": "%1$s在试图逃离%3$s时被%2$s击杀了",
```
请注意参照https://zh.minecraft.wiki/w/%E6%AD%BB%E4%BA%A1%E6%B6%88%E6%81%AF
1.12（不含）以上版本中，通常情况下半角字符与全角字符之间不留空格